Docker/centralserver/files/cs-entrypoint.sh

```diff
@@ -33,7 +33,8 @@ if [ "$INSTALLED_VERSION" == "$PACKAGED_VERSION" ]; then
         sleep 1
         echo "$PACKAGED_VERSION" >/etc/xroad/version
     fi
-    if [ ! -f /home/ca/CA/.init ]; then
+    # Initialize TEST-CA if it has not been initialized yet and the directory exists
+    if [ -d /home/ca ] && [ ! -f /home/ca/CA/.init ]; then
```

Comment:
Here this is needed, because [this CS image](https://github.com/nordic-institute/X-Road/blob/90093ea21af340cee10a5179d14c6fefec5a4184/Docker/centralserver/Dockerfile#L52-L64) contains the Test CA too.